tools/topology/topology2/include/components/gain.conf

```diff
@@ -154,4 +154,5 @@ Class.Widget."gain" {
 	curve_type		"linear"
 	curve_duration		10
 	init_value		0x7fffffff
+	num_audio_formats 1
```

Comment:
can you set this in the right pipeline conf file instead then?

Reply:
> can you set this in the right pipeline conf file instead then?

Indeed, there is no formats defined in gain.conf. So removing it from gain.conf and setting it in pipeline where define available formats makes more sense.